index.html

```
@@ -16,8 +16,42 @@
+                <a
+                    href="#"
+                    class="logo navbar__logo"
+                    aria-label="Travel Point Logo"
+                    ><span class="icon-logo"></span
+                ></a>
+                <a
+                    href="#"
+                    aria-label="Toggle navigation"
+                    aria-expanded="false"
+                    class="navbar__hamburger"
+                    aria-label="Menu"
```

HTMLHint (1.5.0)

[error] Duplicate of attribute name [ aria-label ] was found.

(attr-no-duplication)